### DISCUSSION TITLE
Social-like scrolling mode

### DISCUSSION THREAD

**Discussion by wiersgallak** [mattermost/community#52] [Moonshot Ideas]
Sometimes I want to just scroll through my channels like I do my social media feeds and catch up without having to click into each channel.  This would be especially useful on mobile when I have a few minutes between running errands and waiting for something.  I want to more passively consume content and be able to click into things to reply or engage further.

**Comment by esethna** [comment]
Yep! I think this should fit in with the concept of messaging home screen: https://github.com/mattermost/community/discussions/62

**Reply by esethna** [reply]
The mobile aspect of this is interesting and we should think about that when designing the home screen
